### ISSUE TITLE
TypeError: 'NoneType' object has no attribute '__getitem__'

### ISSUE THREAD

**Issue by edsu** [kennethreitz/pipenv#157]
I can install other things (e.g. requests) but when installing feedparser I see this:

```
(mithcast)bash-3.2$ pipenv install feedparser
Traceback (most recent call last):
  File "/usr/local/bin/pipenv", line 11, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/pipenv/cli.py", line 610, in install
    new_name = proper_case(old_name)
  File "/usr/local/lib/python2.7/site-packages/pipenv/cli.py", line 499, in proper_case
    return parse_download_fname(collected[-1])[0]
  File "/usr/local/lib/python2.7/site-packages/pipenv/cli.py", line 270, in parse_download_fname
    name = r['name']
TypeError: 'NoneType' object has no attribute '__getitem__'
```

**Comment by nateprewitt** [comment]
Thanks for opening this ticket @edsu, could you provide your current Pipfile and the version of pipenv you're using?

**Comment by nateprewitt** [comment]
Sorry @edsu, I'm looking at this again and this is definitely an incarnation of #90 that has since been patched in 3.2.6. The code in the stacktrace no longer exists. Please update your version of pipenv but feel free to reopen this if you're still experiencing a problem.

**Comment by edsu** [comment]
Thanks, upgrading fixed it! Thanks for all your work on pipenv, lovin' it.
